src/auth/schemas/user.schema.ts

```
@@ -0,0 +1,41 @@
+import { HydratedDocument, Types } from 'mongoose';
+
+export type UserDocument = HydratedDocument<User>;
+
+@Schema({ collection: 'users' })
+export class User {
+  _id: Types.ObjectId; 
+
+  @Prop({ required: true })
+  firstName: string;
+
+  @Prop({ required: function() {
```

ESLint

[error] Replace `·required:·function` with `⏎····required:·function·`

(prettier/prettier)